homeassistant/components/miele/sensor.py

```diff
@@ -208,6 +209,27 @@ class MieleSensorDefinition:
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
     ),
+    MieleSensorDefinition(
+        types=(
+            MieleAppliance.WASHING_MACHINE,
+            MieleAppliance.WASHING_MACHINE_SEMI_PROFESSIONAL,
+            MieleAppliance.TUMBLE_DRYER,
+            MieleAppliance.TUMBLE_DRYER_SEMI_PROFESSIONAL,
+            MieleAppliance.DISHWASHER,
+            MieleAppliance.WASHER_DRYER,
+        ),
+        description=MieleSensorDescription(
+            key="energy_forecast",
+            translation_key="energy_forecast",
+            value_fn=lambda value: value.energy_forecast * 100
```

Comment:
Use parenthesis to improve readability of this multiline lambda
```suggestion
            value_fn=(
                lambda value: value.energy_forecast * 100
                if value.enery_forecast is not None
                else None
                ),
```